Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -362,4 +360,14 @@ - (void)searchLanguageDidChangeWithNotification:(NSNotification*)note {
     [self configureSearchViewController];
 }
 
+#pragma mark - UINavigationControllerDelegate
+
+- (void)navigationController:(UINavigationController*)navigationController
```

Comment:
@coreyfloyd on second thought i can just use the logic that's there now: show the toolbar if the view controller has items, otherwise hide it